corehq/apps/reports/models.py

```diff
@@ -578,27 +579,33 @@ def by_domain_and_owner(cls, domain, owner_id, stale=True, **kwargs):
         return result
 
     @property
+    @memoized
     def all_recipient_emails(self):
         # handle old documents
         if not self.owner_id:
             return [self.owner.get_email()]
 
         emails = []
-        if self.send_to_owner:
-            if self.owner.is_web_user():
-                emails.append(self.owner.username)
-            else:
-                email = self.owner.get_email()
-                try:
-                    validate_email(email)
-                    emails.append(email)
-                except Exception:
-                    pass
+        if self.send_to_owner and self.owner_email:
+            emails.append(self.owner_email)
         emails.extend(self.recipient_emails)
         return emails
 
     @property
     @memoized
+    def owner_email(self):
+        if self.owner.is_web_user():
+            return self.owner.username
```

Comment:
can you explain this logic? i would think for web users we would also use email and then fallback to username.

i think the main reason these would be different is if you have since changed your email since you setup your account, which presumably means you want us to contact you on that email?